inductiva/fluids/scenarios/wind_tunnel/post_processing.py

```diff
@@ -0,0 +1,86 @@
+"""Visualization processing of WindTunnel scenario.
+
+This class implements various visualization capabilities for
+the WindTunnel scenario. Namely:
+    - Pressure over object; 
+    - Cutting plane;
+    - StreamLines.
+
+Currently, we only support the OpenFOAM simulator.
+"""
+from functools import singledispatchmethod
+
+import os
+import pyvista as pv
+
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import OpenFOAM
+from inductiva.types import Path
+from inductiva.utils.visualization import Object
+
+
+class WindTunnelSimulationOutput:
```

Comment:
This class seems to be just a data holder (which is actually fine). I don't see any visualization methods here, are they? Or are they being read from data? 
This is not clear...